decoimpact/business/entities/rules/time_aggregation_rule.py

```diff
@@ -157,6 +156,15 @@ def _perform_operation(self, aggregated_values: DataArrayResample) -> _xr.DataAr
         elif self._operation_type in period_operations:
             result = aggregated_values.reduce(self.analyze_groups, dim="time")
 
+        elif self._operation_type is TimeOperationType.STDEV:
+            result = aggregated_values.std()
+
+        elif self._operation_type is TimeOperationType.QUANT10:
+            result = aggregated_values.quantile(0.1).drop_vars("quantile")
+
+        elif self._operation_type is TimeOperationType.QUANT90:
```

Comment:
these are now hard-coded for 10 and 90%. But if a user wants 99, or 80% or whatever, we need to recode this. Can we make this generic? Where the percentage is taken from the input yaml, and we only need one rule to handle this?

Reply:
especially since there is a value in the rule itself -> could be a variable

Reply:
good idea, I will try to implement that